iFixit/Bootstrap-Form-Builder
# Drag and Drop: fix bug with dropping snippets in a list

Starting with the second component, form elements can only be dropped
into the second position from the top. Same thing happens when reordering
components within a form.

As far as I can tell from looking at the code this has been broken for a
long time.

QA
----
I adjusted some configuration around which vertical slot a form element
is dropped into based on where the cursor is hovering. Play around with
drag and drop a bit and make sure I didn't break anything else while I
was fixing this bug.

Connects https://github.com/iFixit/dozuki/issues/2157
